### PR TITLE
Commiting work to show diego

### PR DESCRIPTION
**Pull request by deadlybutter**

### CODE REVIEW

**Comment by deadlybutter** [comment]
@weerd if you wanna take a look :eyes:

File: app/Http/routes.php [hunk 1]
```diff
@@ -27,6 +27,8 @@
     Route::resource('competitions', 'CompetitionsController');
 
     // Waiting rooms routes.
+    Route::get('waitingrooms/{waitingrooms}/split', 'WaitingRoomsController@showSplitForm')->name('split');
+    Route::post('waitingrooms/{waitingrooms}/split', 'WaitingRoomsController@split')->name('split');
```
Comment (weerd) [inline app/Http/routes.php]: I've never seen it like you have it... only like this:

```
Route::get('waitingrooms/{waitingrooms}/split', ['as' => 'waitingrooms.split', 'uses' => 'WaitingRoomsController@showSplitForm']);
```

Comment (deadlybutter) [inline app/Http/routes.php]: hmm ill try that then

Comment (deadlybutter) [inline app/Http/routes.php]: nope @weerd still getting this error `Missing required parameters`

Comment (deadlybutter) [inline app/Http/routes.php]: fixed the route to look like that for both the post & get, also updated the views

**Pull request closed without merging**